packages/objection/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './interpreters';
```

Comment:
Very nice! I'm very happy that you used the structure from existing ucast module :+1: